client/src/components/Profile/Profile.js

```diff
@@ -0,0 +1,192 @@
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+const Profile = () => {
+  const navigate = useNavigate();
+
+  const state = {
+    user: useSelector((state) => state.user),
+    previewSource: '',
+    message: '',
+    fileTypes: ['jpg', 'png', 'jpeg', 'image/jpg', 'image/png', 'image/jpeg'],
+  };
+
+  // const user = useSelector((state) => state.user);
+
+  const changeAvatar = () => {
```

Comment:
All of this logic I will also utilize to integrate into my profile page so that I can get the upload avatar working.